apps/backoffice-v2/src/lib/blocks/components/PDFViewerCell/PDFViewer.tsx

```diff
@@ -0,0 +1,7 @@
+import { TPDFViewerCell } from '@/lib/blocks/components/PDFViewerCell/interfaces';
+import { FunctionComponent } from 'react';
+
+export const PDFViewerCell: FunctionComponent<TPDFViewerCell> = ({ props, value }) => {
+  const { width, height } = props;
+  return value ? <iframe src={`${value}#navpanes=0`} width={width} height={height} /> : null;
```

Comment:
I think an early return would last longer and will be quicker to understand what happens when there is no value.

```suggestion
  if (!value) {
      return;
  }

  return <iframe src={`${value}#navpanes=0`} width={width} height={height} />;
```